packages/0x.js/src/contract_wrappers/contract_wrapper.ts

```diff
@@ -52,8 +52,7 @@ export class ContractWrapper {
             throw new Error(ZeroExError.SubscriptionNotFound);
         }
         if (!_.isUndefined(err)) {
-            const callback = this._filterCallbacks[filterToken];
-            callback(err, undefined);
+            this._filterCallbacks[filterToken](err, undefined);
```

Comment:
I actually prefer us to first set it to a variable called `callback`, for readability.